pom.xml

```diff
@@ -287,6 +286,11 @@
         <artifactId>junit</artifactId>
         <version>1.3</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-lang3</artifactId>
+        <version>3.7</version>
+      </dependency>
```

Comment:
That addition alone looks very fishy. Did you mean to add it with `test` scope?

Reply:
it is needed for enforcer.RequireUpperBoundDeps